File: samples/clientpetstorecsharp/src/main/csharp/io/swagger/Model/Pet.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;



namespace io.swagger.Model {
  public class Pet {
    

    
    public long? Id { get; set; }

    

    
    public Category Category { get; set; }

    

    
    public string Name { get; set; }

    

    
    public List<string> PhotoUrls { get; set; }

    

    
    public List<Tag> Tags { get; set; }

    

    /* pet status in the store */
    
    public string Status { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Pet {\n");
      
      sb.Append("  Id: ").Append(Id).Append(Environment.NewLine);
      
      sb.Append("  Category: ").Append(Category).Append(Environment.NewLine);
      
      sb.Append("  Name: ").Append(Name).Append(Environment.NewLine);
      
      sb.Append("  PhotoUrls: ").Append(PhotoUrls).Append(Environment.NewLine);
      
      sb.Append("  Tags: ").Append(Tags).Append(Environment.NewLine);
      
      sb.Append("  Status: ").Append(Status).Append(Environment.NewLine);
      
      sb.Append("}");
      sb.Append(Environment.NewLine);
      return sb.ToString();
    }
  }
  
  
}

File: samples/client/petstore/gatling/src/gatling/scala/io/swagger/client/model/Pet.scala

package io.swagger.client.model

import io.swagger.client.model.Category
import io.swagger.client.model.List
import io.swagger.client.model.Tag

case class Pet (
    id: Option[Long],
    category: Option[Category],
    name: String,
    photoUrls: List[String],
    tags: Option[List[Tag]],
    /* pet status in the store */
    status: Option[String]
)
object Pet {
    def toStringBody(id: Object, category: Object, name: Object, photoUrls: Object, tags: Object, status: Object) =
        s"""
        | {
        | "id":$id,"category":$category,"name":$name,"photoUrls":$photoUrls,"tags":$tags,"status":$status
        | }
        """.stripMargin
}
